pyaedt/hfss.py

```diff
@@ -459,8 +460,24 @@ def create_linear_count_sweep(self, setupname, unit, freqstart, freqstop, num_of
 
         Returns
         -------
-        SweepHFSS
-            Sweep object.
+        :class:`pyaedt.modules.SetupTemplates.SweepHFSS` or bool
+            Sweep object if successful. ``False`` if unsuccessful.
+
+        Examples
+        --------
+
+        Create a setup named ``'LinearCountSetup'``.
+        Create a linear count sweep named ``'LinearCountSweep'`` for the analysis setup
+        created above.
```

Comment:
```suggestion
        Create a setup named ``'LinearCountSetup'`` and use it in a linear count sweep
        named ``'LinearCountSweep'``.
```
I think this reads better.  Please use a similar description in `create_frequency_sweep`

Reply:
Sure, I will change all the similar comments in my other examples.